<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherInterface</class>
 <widget class="QWidget" name="TeacherInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>57</y>
     <width>261</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="analyzeScorePushButton">
      <property name="text">
       <string>班级成绩分析</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selfMakeExamPushButton">
      <property name="text">
       <string>自行出卷并下发</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="teacherInfoLabel">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>31</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>教师：xxxx  班级号:xxxx</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeInfoPushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>259</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>更改信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
